content/product-engineering/engineering/enablement/frontend-platform/index.md

```diff
@@ -19,29 +19,29 @@ The Frontend Platform team (part of the [Enablement](../index.md) org) defines a
 
 1. Frontend platform:
 
-   - Creating and maintaining the Wildcard Component Library.
-   - Owning the Sourcegraph web tech stack, tools, and patterns.
-   - Documentation and training material enables product teams and new hires to learn how to do web development at Sourcegraph quickly.
-   - Define and maintain how we test and deploy frontend code.
-   - Ensuring an efficient and reliable frontend CI pipeline.
-   - Track, measure, and improve cross-cutting frontend metrics like bundle size, [Web vitals](https://web.dev/vitals/), etc.
+   - Creating and maintaining the Wildcard Component Library
+   - Owning the Sourcegraph web tech stack, tools, and patterns
+   - Providing documentation and training material that enables product teams and new hires to learn how to do web development at Sourcegraph quickly
+   - Defining and maintaining how we test and deploy frontend code
+   - Ensuring an efficient and reliable frontend CI pipeline
+   - Tracking, measuring, and improving cross-cutting frontend metrics like bundle size, [Web vitals](https://web.dev/vitals/), etc.
 
 2. The core user experience of the Sourcegraph product:
    - Accessibility, navigation, and information hierarchy
    - Performance and efficiency of the core UI
    - All code browsing and code host-like views
    - Code syntax highlighting
```

Comment:
I don't think we own these anymore:
- We own some code browsing reviews (like repository pages).
- I don't think we own any code host views? That makes me think of the extensibility team.
- I also don't think we own syntax highlighting anymore. I think we discussed this but I can't remember what team does.

Reply:
We do own the user settings and site admin pages so we should include that

Reply:
👍 Do the diff views fall under "Code browsing and code host-like views"? (And you're right about syntax highlighting, which I think is Code Intel now.)

Reply:
I guess so! I think it's tricky to summarise, maybe this should just be "Some areas of code browsing".

We could also link to https://handbook.sourcegraph.com/product-engineering/engineering/engineering_ownership for a more detailed break down